messages/action-summary-viewer.md

```diff
@@ -0,0 +1,15 @@
+# common.summary-header
+
+Summary
+
+# common.logfile-location
+
+Additional log information written to:
```

Comment:
Copied from `run-summary-viewer.md`. Named `common.blah` to allow for porting the existing `RunSummaryViewer` implementation to this new style.